src/lib/keccak.ts

```diff
@@ -1,31 +1,23 @@
 import { Field } from './field.js';
```

Comment:
Actually, does this fit into `src/lib/gadgets` better than `src/lib`? Similar to ECDSA

Reply:
It probably does! IIRC, I put it there before ECDSA was done, but it should probably live wherever that lives!

Reply:
Agreed, I can move it in the later PR!